helm-charts/public-nginx_values.yaml

```diff
@@ -15,16 +29,24 @@ controller:
    - name: mattermost-cache
      mountPath: /cache/mattermost
 
-  extraVolumes: 
+  extraVolumes:
   ## Additional volumes to the controller pod.
    - name: cache
      emptyDir: {}
    - name: mattermost-cache
      emptyDir: {}
 
-  config: 
+  config:
     http-snippet: |
       proxy_cache_path /cache/nginx levels=1:2 keys_zone=mattermost_test_cache:10m max_size=10g inactive=120m use_temp_path=off;
       proxy_cache_path /cache/mattermost levels=1:2 keys_zone=mattermost_cache:10m max_size=10g inactive=120m use_temp_path=off;
     keep-alive: "3600"
     proxy-max-temp-file-size: "0"
+    # Will add custom configuration options to Nginx https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/
+    ssl-redirect: "false"
```

Comment:
@cpanato where are we handling the redirect now?

Reply:
i will make a quick test on that

Reply:
tested with those configs and redirect works as well